src/Components/FormCreation/CreateEditForm.js

```diff
@@ -81,12 +82,17 @@ function CreateEditForm() {
   const [showFormSettings, setShowFormSettings] = useState(false);
   const [showMoveSectionsDialog, setShowMoveSectionsDialog] = useState(false);
   const [activeSection, setActiveSection] = useState(0);
-  const programId = appUser.currentProgram;
+  //const programId = appUser.currentProgram;
```

Comment:
`programId` looks like it's being passed in as a prop now, so can probably remove this.

Reply:
Good catch